lib/hls/hls_parser.js

```diff
@@ -2971,7 +2971,7 @@ shaka.hls.HlsParser = class {
     const stream = this.makeStreamObject_(streamId, codecs, type,
         languageValue, primary, name, channelsCount, closedCaptions,
         characteristics, forced, sampleRate, spatialAudio);
-    stream.encrypted = encrypted;
+    stream.encrypted = encrypted && !aesEncrypted;
```

Comment:
Perhaps the description in the externs for the `encrypted` value should be updated if it does not count aes encryption?

Reply:
Done!